include/fiction/algorithms/physical_design/design_sidb_gates.hpp

```diff
@@ -257,7 +345,7 @@ class design_sidb_gates_impl
      * @param cell_indices A vector of indices of cells to be added to the skeleton layout.
      * @return A copy of the original layout (`skeleton_layout`) with SiDB cells added at specified indices.
      */
-    [[nodiscard]] Lyt skeleton_layout_with_canvas_sidbs(const std::vector<std::size_t>& cell_indices) noexcept
+    [[nodiscard]] Lyt skeleton_layout_with_canvas_sidbs(const std::vector<std::size_t>& cell_indices) const noexcept
```

Comment:
From the function name alone it is unclear what is performed. Should it be `get_skeleton_layout_...` or maybe `create_skeleton_layout_...`? Or something else altogether?